install/centos/README.md

```diff
@@ -226,27 +212,36 @@ Remove any other Ruby build if it is still present:
 Download Ruby and compile it:
 
     mkdir /tmp/ruby && cd /tmp/ruby
-    curl --progress ftp://ftp.ruby-lang.org/pub/ruby/2.1/ruby-2.1.2.tar.gz | tar xz
-    cd ruby-2.1.2
+    curl --progress ftp://ftp.ruby-lang.org/pub/ruby/2.1/ruby-2.1.10.tar.gz | tar xz
+    cd ruby-2.1.10
     ./configure --disable-install-rdoc
     make
     make prefix=/usr/local install
 
 Install the Bundler Gem:
 
-    gem install bundler --no-doc
+    gem install bundler --no-ri --no-rdoc
```

Comment:
I know, but Ubuntu guide uses --no-ri --no-rdoc and I wanted to be as close to possible to that one. However Ubuntu guide probably should be updated to use --no-doc.